spyder/plugins/explorer/widgets.py

```diff
@@ -181,6 +181,38 @@ def icon(self, icontype_or_qfileinfo):
                 return icon
 
 
+class ColorModel(QFileSystemModel):
+    """FileSystemModel providing a color-code for different commit-status."""
+    def __init__(self, *args, **kwargs):
+        super(ColorModel, self).__init__(*args, **kwargs)
+        self.vcs_state = None
+        self.color_array = [QColor("#ff0000"), QColor("#555555"),
+                            QColor("#0099ff"), QColor("#00ff00"),
+                            QColor("#ffffff")]

```

Comment:
I need to replace those with values from the styles. Otherwise, this will cause problems in light themes.